src/react/hooks/use-messages.ts

```diff
@@ -101,6 +106,7 @@ export const useMessages = (params?: UseMessagesParams): UseMessagesResponse =>
     [room],
   );
   const get = useCallback((options: QueryOptions) => room.messages.get(options), [room]);
+  const update = useCallback(room.messages.update.bind(room.messages), [room])
```

Comment:
Is there a need to use bind here, or can we keep it consistent with the other two callbacks?

Reply:
updated to be consistent